lib/ansible/modules/cloud/ovirt/ovirt_host.py

```diff
@@ -367,6 +367,16 @@ def failed_state_after_reinstall(self, host, count=0):
         return False
 
 
+def remove_host(host_module, host, max_count=5, timeout=20):
+    for count in range(max_count + 1):
+        try:
+            return host_module.remove()
+        except Exception as e:
+            if count == max_count:
```

Comment:
Please pass wait module parameter, and do no try to retry the removal in case wait is false.